definitions/mobile.d.ts

```diff
@@ -85,8 +85,8 @@ declare module Mobile {
 	}
 
 	interface IAndroidDebugBridge {
-		executeCommand(...args: string[]): IFuture<any>;
-		executeShellCommand(...args: string[]): IFuture<any>;
+		executeCommand(args: string[]): IFuture<any>;
+		executeShellCommand(args: string[]): IFuture<any>;
```

Comment:
If i leave it as-was, the compiler complains with `error TS2345: Argument of type 'string[]' is not assignable to parameter of type 'string'.` I am open to suggestions how else to fix it.